tenable/sc/reports_definition.py

```diff
@@ -0,0 +1,34 @@
+'''
+Reports Definition
+============
+
+The following methods allow for interaction into the Tenable.sc
+:sc-api:`ReportDefinition <ReportsDefinition.html>` API.
+
+Methods available on ``sc.reports_definition``:
+
+.. rst-class:: hide-signature
+.. autoclass:: ReportsDefinitionAPI
+    :members:
+'''
+from .base import SCEndpoint
+
+class ReportsDefinitionAPI(SCEndpoint):
```

Comment:
Should be using a more correct name that conforms to the rest of the package

```suggestion
class ReportDefinitionsAPI(SCEndpoint):
```